test/partitioning.jl

```diff
@@ -196,7 +196,31 @@
   end
 
   @testset "SpatialPredicatePartitioner" begin
-    # TODO
+    g = RegularGrid{Float64}(10,10)
+    R = 5.0
+    SPred1(x,y) = norm(x-y) < R
+
+    p = partition(g, SpatialPredicatePartitioner(SPred1))
+    s = subsets(p)
+    testArray = sample(s)
+    @test (norm(sample(testArray)-sample(testArray)) < R) # --> I already know there is an error here. What I'm trying to check is if coordinates of the sampled point agrees with restriction. i.e. I need to get the coordinates of points in RegularGrid. Could you please help me in this assert?
```

Comment:
Here you are confusing the index of the location with the coordinates of the location. The subsets returned by a partitioner are just indices of the locations in the domain. So when you `sample(testArray)` you are getting an integer and not a vector of coordinates. Please fix the test accordingly.

Reply:
yeah! that's what I tried to say in
> I need to get the coordinates of points in RegularGrid

Is there a way to retrieve point coordinates from its integer index?

Reply:
I belive that `getindex` method does what I need. Anyway, I changed strategy and commited another test inplace.